namespaces/nsenter/nsenter.c

```diff
@@ -151,6 +151,15 @@ void nsenter()
 			exit(1);
 		}
 	}
+	// read continue signal from fd 3. the signal is 1
+	// this is so that the parent can put the proces inside correct cgroups
+	// before it has a chance to fork
+	unsigned char s;
+	if (read(3, &s, 1) != 1 || s != '1') {
+		fprintf(stderr,
+			"nsenter: Failed to get signal from parent: %m\n");
```

Comment:
iirc, `%m` is the same as `%s` with `strerror(errno)`. I'm happy to revert it back though.